index.html

```diff
@@ -26,6 +26,223 @@ <h1>Hey There. I’m Alex</h1>
           </ul>
         </div>
     </header>
+
+    <div class="container">
+        <div class="works">
+            <div class="works-title">
+                <h1>My Recent Works</h1>
+                <div class="line">
+                </div>
+            </div>
+            <div class="grid-container">
+                <div class="grid-item">
+                    <div class="image_intro">
+                      <img src="imgs/Img_Placeholder.svg" alt="intro img">
+                    </div>
+                    <div class="multi-post">
+                        <h2>Multi-Post Stories</h2>
+                        <p>
+                            A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.
+                        </p>
+                        <div class="button">
+                            <div class="flex-container">
+                                <div><button type="button">css</button></div>
+                                <div><button type="button">html</button></div>
+                                <div><button type="button">Bootstrap</button></div> 
+                                <div><button type="button">Ruby</button></div>  
+                              </div>
```

Comment:
- [ ] The list of project technologies (e.g., "JavasScript", "HTML" and "CSS") are a list `ul` and each of them a `li`, please kindly change it :grin: :grin:, also make the change for all part that you have the project technologies.